packages/ihe-gateway-sdk/src/models/patient-discovery/patient-discovery-responses.ts

```diff
@@ -1,21 +1,47 @@
 import * as z from "zod";
 import {
   baseResponseSchema,
+  BaseResponse,
   baseErrorResponseSchema,
   externalGatewayPatientSchema,
   XCPDGatewaySchema,
 } from "../shared";
 
+// TODO: Test that this schema is correct for the patientResource in the IHE response
```

Comment:
Is this a left-over or are you still testing this?

I was going to ask about name, to make sure the array is always present, but we should test all fields yeah

Reply:
Yea, I was still testing it. Fixed in this PR - https://github.com/metriport/metriport/pull/1747/files